scripts/package/hooks/after-install/hook

```diff
@@ -0,0 +1,67 @@
+#! /usr/bin/env bash
+
+# This script is called by a system package manager to request on-site
+# setup on a host, after the package itself is installed.
```

Comment:
What does "on-site setup on a host after the package itself is installed" mean?

Reply:
It means `apt`, `yum`, etc. passed control to this script, to do setup steps the package manager itself cannot do.

Reply:
Happy to phrase differently if this is confusing.